dashboard/app/controllers/api/v1/pd/fit_weekend_registrations_controller.rb

```diff
@@ -3,8 +3,14 @@ class Api::V1::Pd::FitWeekendRegistrationsController < Api::V1::Pd::FormsControl
   load_and_authorize_resource :application, class: 'Pd::Application::ApplicationBase', id_param: :applicationId
 
   def new_form
-    FIT_WEEKEND_REGISTRATION_CLASS.new(
+    @fit_registration = FIT_WEEKEND_REGISTRATION_CLASS.new(
       pd_application_id: @application.id
     )
   end
+
+  def on_successful_create
+    return unless @fit_registration.accepted?
+
+    @application.enroll_user
```

Comment:
```rb
@application.enroll_user if @fit_registration.accepted?
```

Reply:
I always forget about the one liners! thanks!